### PR TITLE
Changes Travis script to also run make doctest

### PR DESCRIPTION
**Pull request by SrNetoChan**
### Description

Now, that we have some tests with Python doctest, we can/should make travis test them regularly.

I am not sure I know what doing here, so this is probably more a call for action than a proper PR.

Goal:

Make travis test the python snippets

Ticket: #

- [ ] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

**Comment by SrNetoChan** [comment]
ok, as is, Travis will complain too much. If I recall, if there are no testcode directives, sphinx will try to test every piece of code that he can find...  we need to merge the CRS on that I think it's ready.

**Comment by SrNetoChan** [comment]
ohhh, yes I forgot about that one...

File: .travis.yml [hunk 1]
```diff
@@ -50,6 +50,7 @@ install:
 
 script:
   - make html
+  - make doctest
```
Comment (ghtmtt) [inline .travis.yml]: maybe:

`make -f docker.mk doctest`?

**Comment by SrNetoChan** [comment]
Looks like @arnaud-morvan already created a PR with the needed fixes. And @ghtmtt you were right. :)

**Comment by ghtmtt** [comment]
> Looks like @arnaud-morvan already created a PR with the needed fixes. And @ghtmtt you were right. :)

but I was missing something in the makefile ;)

**Comment by SrNetoChan** [comment]
One step at the time...

**Pull request closed without merging**